doc/nrf/ug_matter_configuring_protocol.rst

```diff
@@ -0,0 +1,75 @@
+.. _ug_matter_configuring_protocol:
+
+Configuring Matter in |NCS|
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page describes what is needed to start working with Matter in the |NCS|.
+
+.. _ug_matter_configuring_requirements:
+
+Required components for Matter network
+**************************************
+
+The Matter network requires the following components to properly operate:
+
+* Matter controller - configured either on PC or mobile
+* Thread Border Router - required by the Matter controller
+
+For information about how to configure these components, read :ref:`ug_matter_configuring`.
+
+.. _ug_matter_configuring_mandatory:
+
+Mandatory configuration
+***********************
+
+To use the Matter protocol, enable the :kconfig:`CONFIG_CHIP` Kconfig option.
+Setting this option enables the Matter protocol stack and other associated Kconfig options, including :kconfig:`CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the discovery of the Matter device using DNS-SD.
+
+After that, you have to set the :kconfig:`CONFIG_CHIP_PROJECT_CONFIG` Kconfig options and define the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
```

Comment:
```suggestion
After that, you can set the :kconfig:`CONFIG_CHIP_PROJECT_CONFIG` Kconfig option and define the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
```
As this is optional (but recommended). In theory one can use Matter default settings.